backend/serve.go

```diff
@@ -131,6 +131,9 @@ func defaultGRPCMiddlewares(opts ServeOpts) []grpc.ServerOption {
 }
 
 // Serve starts serving the plugin over gRPC.
+//
+// Deprecated: Serve exists for historical compatibility
+// and might be removed in a future version. Please migrate to use [Manage] instead.
 func Serve(opts ServeOpts) error {
```

Comment:
I guess the plan is to just eventually unexport this since it's used by `Manage`?

Reply:
Yeah probably